.github/dependabot.yml

```diff
@@ -0,0 +1,64 @@
+version: 2
+updates:
+  # Enable version updates at root 
+  - package-ecosystem: "npm"
+    # Look for `package.json` and `lock` files in the `root` directory
+    directory: "/"
+    # Check the pnpm registry for updates every day (weekdays)
+    schedule:
+      interval: "daily"
+      time: "03:00"
+      timezone: "Europe/Berlin"
+    labels:
+      - "npm dependencies"
+      - "root"
+    commit-message:
+      prefix: fix
```

Comment:
Is this on purpose? this would bump our version every time we have an update on any dependency...

the docs say
``` By default, Dependabot attempts to detect your commit message preferences and use similar patterns. Use the commit-message option to specify your preferences explicitly.```
and I would go with this solution first... 

Not every dependency update is a `fix` but most of the would be a `chore`. In the CMS project we use renovate and its detecting (depending on the update) if its a fix or a chore.

Reply:
Ah, I see. Switched to chore.

Reply:
Sorry, maybe not clear enough. If you leave out the whole config element, it will (hopefully) set the prefix itself. 

As it can be both (fix or chore), we cannot specify it in the config.

Reply:
Ahh, I see. I totally misread that. Thanks for the re-clarification.